src/locales/en.json

```diff
@@ -1,19 +1,24 @@
 {
-  "DEFAULT_ERRORS": {
-    "AUTHENTICATION_FAILED": "Please login to access this resource.",
-    "INTERNAL_SERVER_ERROR": "Internal server error. Please try again later.",
-    "LOGIN_AUTHENTICATION_FAILED": "Invalid credentials. Please check your email and password and try again.",
-    "RESOURCE_NOT_FOUND": "Requested resource not found.",
-    "VALIDATION_FAILED": "Validation failed. Please modify the request and try again."
-  },
-  "VALIDATION_ERRORS": {
-    "INVALID_EMAIL": "Please specify valid email address",
-    "INVALID_FIRSTNAME": "Please specify first name.",
-    "INVALID_LASTNAME": "Please specify last name.",
-    "PASSWORD_MISSING": "Please provide a password.",
-    "DUPLICATE_EMAIL": "The email you provided is already taken. Please provide a different one."
-  },
-  "MESSAGES": {
-    "HEALTHCHECK": "OK"
-  }
-}
+	"DEFAULT_ERRORS": {
+		"AUTHENTICATION_FAILED": "Please login to access this resource.",
+		"INTERNAL_SERVER_ERROR": "Internal server error. Please try again later.",
+		"LOGIN_AUTHENTICATION_FAILED": "Invalid credentials. Please check your email and password and try again.",
+		"RESOURCE_NOT_FOUND": "Requested resource not found.",
+		"VALIDATION_FAILED": "Validation failed. Please modify the request and try again."
+	},
+	"VALIDATION_ERRORS": {
+		"INVALID_EMAIL": "Please specify valid email address",
+		"INVALID_FIRSTNAME": "Please specify first name.",
+		"INVALID_LASTNAME": "Please specify last name.",
+		"PASSWORD_MISSING": "Please provide a password.",
+		"DUPLICATE_EMAIL": "The email you provided is already taken. Please provide a different one.",
+		"INVALID_TITLE": "VALIDATION_ERRORS.INVALID_TITLE",
+		"DUPLICATE_ENTRY": "VALIDATION_ERRORS.DUPLICATE_ENTRY",
+		"INVALID_ID": "VALIDATION_ERRORS.INVALID_ID"
+	},
+	"MESSAGES": {
+		"HEALTHCHECK": "OK"
+	},
+	"Cast to ObjectId failed for value \"2114071\" at path \"_id\" for model \"TodoItem\"": "Cast to ObjectId failed for value \"2114071\" at path \"_id\" for model \"TodoItem\"",
+	"The specified todo ID is not a valid one. Please provide a valid one.": "The specified todo ID is not a valid one. Please provide a valid one."
+}
```

Comment:
For this file:
- Spacing here is changed and does not seems right
- Please make sure we leave a new line by EOF